client/ayon_core/addons/applications/ayon_applications/addon.py

```diff
@@ -11,6 +11,10 @@
 class ApplicationsAddon(AYONAddon, IPluginPaths):
     name = "applications"
 
+    def initialize(self, settings):
+        # TODO remove when addon is removed from ayon-core
+        self.enabled = self.name in settings
```

Comment:
How come this is needed? Is it because addons present in ayon core are currently **always** loaded no matter what?

Reply:
Yes.